Update milestone information in genesis file (#987)
hyperledger/besu-docs: docs/Reference/Config-Items.md

    {
      "config": {
        ...
        "londonBlock": 0,
        ...
      },
    }
    ```

> !!! tip

Reviewer: ```suggestion
!!! note
```

Author: addressed

>     When specifying the milestone block for private networks, we recommend specifying the latest

Reviewer 2: We recommend specifying the latest milestone block for private networks.
or
When starting a new private network, we recommend specifying the latest milestone

Author: addressed

>     It is implied this includes the preceding milestones.
>     Specifying the latest milestone ensures you are using the most up-to-date protocol and have access to the most recent opcodes.

Reviewer: ```suggestion
    This ensures you're using the most up-to-date protocol and have access to the most recent opcodes.
```

Author: addressed
